doctrine/multiple_entity_managers.rst

```diff
@@ -204,27 +208,25 @@ entity manager to persist and fetch its entities.
 
 The same applies to repository calls::
 
+    use Doctrine\Common\Persistence\ManagerRegistry;
+
     class UserController extends Controller
```

Comment:
I've been using `Controller` everywhere to ease the transition to the "type" based model. I think if we switch to it too fast, we'll have problems with people doing `$this->get('...')` while reading blog posts, README's, etc and it not working. And there's really not much profit in using `AbstractController`.